code/src/components/Labyrinth.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { continueGame } from 'reducers/start';
+import { End } from './End';
+
+import {
+  DescriptionContainer,
+  TitleContainer,
+  Choices,
+  Button,
+  ChoicesText,
+} from './StyledComponents';
+
+export const Labyrinth = () => {
+  const dispatch = useDispatch();
+
+  const coordinates = useSelector(store => store.start.coordinates);
+  const gameDescription = useSelector(store => store.start.description);
+  const gameActions = useSelector(store => store.start.actions);
```

Comment:
An easier way maybe to deconstruct it: const { description, coordinates, actions } = useSelector(store => store.game.currentPosition)